gitlab/client.py

```diff
@@ -66,14 +71,20 @@ def __init__(
         http_password: Optional[str] = None,
         timeout: Optional[float] = None,
         api_version: str = "4",
-        session: Optional[requests.Session] = None,
         per_page: Optional[int] = None,
         pagination: Optional[str] = None,
         order_by: Optional[str] = None,
         user_agent: str = gitlab.const.USER_AGENT,
         retry_transient_errors: bool = False,
         keep_base_url: bool = False,
+        **kwargs: Any,
     ) -> None:
+        """
+        kwargs:-
+        :param requests.Session session: (optional). Http Requests Session.
+        :param RequestsEngine http_engine: (optional). Engine that will be used
+               to make http requests.
+        """
```

Comment:
Quick note that we use the google style docstrings in this project (https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html). See the class docstring above.

Also, should this be placed along with the class docstring? It looks a bit out of place here on the dunder method. 
See this discussion on SO:
https://stackoverflow.com/questions/1137161/what-is-the-correct-way-to-document-a-kwargs-parameter

Reply:
Thanks @nejch for clarification. I would expected the CI to catch that one. But, not in the scope of this PR.  Pushed a fix. LMK if that is still not in the correct format.